lib/to_do_app/todos/items.ex

```diff
@@ -0,0 +1,18 @@
+defmodule ToDoApp.Todos.Items do
```

Comment:
Change the module name from  
```ToDoApp.Todos.Items```

to 

```ToDoApp.Todos.Item```

It is conventional to keep the module name singular

And also keep the file name as  `item.ex`